.eslintrc.js

```diff
@@ -0,0 +1,82 @@
+module.exports = {
+  root: true,
+  extends: [
+    // these are relics of nimbus, we could definitely simplify + consolidate
+    './config-eslint/base.js',
```

Comment:
this config is kind of out of control but I don't think trying to simplify it in this PR is a good idea given the amount of changes already here